server/package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "build": "npx tsc",
     "start": "node build/index.js",
+    "test": "jest",
```

Comment:
A note in the README about running tests with `npm test` would be good.

Reply:
Agreed.